### ISSUE TITLE
Please include fly.toml to .gitignore

### ISSUE THREAD

**Issue by marcusfpereira** [nightscout/cgm-remote-monitor#7992]
Please incude fly.toml to .gitignore so I it is not uploaded do personal repositories clones.

diff --git a/.gitignore b/.gitignore
index 7b8875e5..8d665b38 100644
--- a/.gitignore
+++ b/.gitignore
@@ -11,6 +11,8 @@ my.env
 my.*.env
 *.pem
 
+fly.toml
+
 static/bower_components/
 .*.sw?
 .DS_Store

**Comment by psonnera** [comment]
@bewest I agree this is an important and simple modification to avoid people exposing sensitive information in their fork by mistake. As discussed before, we recommend people use their own fork to deploy Nightscout instead of this repo.

**Comment by AndyLow91** [comment]
resolved by #8359
